package.json

```diff
@@ -20,125 +21,18 @@
   "author": "Aaron Heckmann <aaron.heckmann+github@gmail.com>",
   "license": "MIT",
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "@typescript-eslint/parser": "^5.30.7",
     "benchmark": "~2.1.4",
     "eslint": "^7.16.0",
-    "mocha": "^9.2.2"
+    "eslint-plugin-mocha-no-only": "^1.1.1",
+    "mocha": "^9.2.2",
+    "typescript": "^4.7.4"
   },
-  "eslintConfig": {
-    "extends": [
-      "eslint:recommended"
-    ],
-    "parserOptions": {
-      "ecmaVersion": 2015
-    },
-    "env": {
-      "node": true,
-      "es6": true
-    },
-    "rules": {
-      "comma-style": "error",
-      "indent": [
-        "error",
-        2,
-        {
-          "SwitchCase": 1,
-          "VariableDeclarator": 2
-        }
-      ],
-      "keyword-spacing": "error",
-      "no-whitespace-before-property": "error",
-      "no-buffer-constructor": "warn",
-      "no-console": "off",
-      "no-multi-spaces": "error",
-      "no-constant-condition": "off",
-      "func-call-spacing": "error",
-      "no-trailing-spaces": "error",
-      "no-undef": "error",
-      "no-unneeded-ternary": "error",
-      "no-const-assign": "error",
-      "no-useless-rename": "error",
-      "no-dupe-keys": "error",
-      "space-in-parens": [
-        "error",
-        "never"
-      ],
-      "spaced-comment": [
-        "error",
-        "always",
-        {
-          "block": {
-            "markers": [
-              "!"
-            ],
-            "balanced": true
-          }
-        }
-      ],
-      "key-spacing": [
-        "error",
-        {
-          "beforeColon": false,
-          "afterColon": true
-        }
-      ],
-      "comma-spacing": [
-        "error",
-        {
-          "before": false,
-          "after": true
-        }
-      ],
-      "array-bracket-spacing": 1,
-      "arrow-spacing": [
-        "error",
-        {
-          "before": true,
-          "after": true
-        }
-      ],
-      "object-curly-spacing": [
-        "error",
-        "always"
-      ],
-      "comma-dangle": [
-        "error",
-        "never"
-      ],
-      "no-unreachable": "error",
-      "quotes": [
-        "error",
-        "single"
-      ],
-      "quote-props": [
-        "error",
-        "as-needed"
-      ],
-      "semi": "error",
-      "no-extra-semi": "error",
-      "semi-spacing": "error",
-      "no-spaced-func": "error",
-      "no-throw-literal": "error",
-      "space-before-blocks": "error",
-      "space-before-function-paren": [
-        "error",
-        "never"
-      ],
-      "space-infix-ops": "error",
-      "space-unary-ops": "error",
-      "no-var": "warn",
-      "prefer-const": "warn",
-      "strict": [
-        "error",
-        "global"
-      ],
-      "no-restricted-globals": [
-        "error",
-        {
-          "name": "context",
-          "message": "Don't use Mocha's global context"
-        }
-      ],
-      "no-prototype-builtins": "off"
-    }
-  }
+  "files": [
```

Comment:
I generally prefer to not ship with `files`, because adding a file but forgetting to add it to `files` is an easy mistake to make.

Is `files` just for unpkg?

Reply:
> Is files just for unpkg?

no, unpkg is to show the selected published "unzipped" version (like manually downloading a version from npm and then "unzipping" it and inspecting it), "files" (or like in mongoose "directories") is used to tell npm / yarn to only package these files into the archive and only publish those files (some files like "LICENSE" and "package.json" are always included)

> I generally prefer to not ship with files, because adding a file but forgetting to add it to files is an easy mistake to make.

this is true, but if the project structure is kept (as in everything in `src` / `lib` / `build` / `dist`), then there should never be a non-working package once properly defined, though if adding files (or renaming) like a changelog, then it could be forgotten

in the case of mpath i had decided to just go with `files` over `directories`, because of the root `index.js`, which (from what i know) could not be easily included in `directories` only would still require `files` for just that

Reply:
aside from that, i also just noticed that #14 has also already added the "files" property